src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/ResolvedProjectReference.xaml

```diff
@@ -40,7 +40,14 @@
 
   <BoolProperty Name="CopyLocalSatelliteAssemblies"
                 Description="Indicates whether the satellite assemblies of the reference target should be copied into this project's output directory."
-                DisplayName="Copy Local Satellite Assemblies" />
+                DisplayName="Copy Local Satellite Assemblies">
+    <BoolProperty.DataSource>
+      <DataSource HasConfigurationCondition="False"
+                  ItemType="ProjectReference"
+                  Persistence="ProjectFile"
```

Comment:
This is an example of adding a property-level data source that persists to the project file on resolved dependencies. Without this, the rule-level data source is inherited and cannot write its value anywhere. There are several instances of this throughout the PR.